airflow/config_templates/config.yml.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string"
+      },
+      "description": {
+        "type": ["string", "null"]
+      },
+      "options": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": ["string", "null"]
+            },
+            "version_added": {
+              "type": ["string", "null"]
+            },
+            "type": {
+              "type": "string"
+            },
+            "example": {
+              "type": ["string", "null"]
+            },
+            "default": {
+              "type": ["string", "null"]
```

Comment:
This should allow numbers too I think

Reply:
Yeah you're right, I think `example` probably should as well.